app/models/image.rb

```diff
@@ -19,6 +22,19 @@ class Image < ActiveRecord::Base
   scoped_search :relation => :operatingsystem, :on => :id, :rename => "operatingsystem", :complete_enabled => false, :only_explicit => true, :validator => ScopedSearch::Validators::INTEGER
   scoped_search :on => :user_data, :complete_value => {:true => true, :false => false}
 
+  def password_changed_changed?
```

Comment:
I had a feeling this isn't the only place where we don't want audits to include passwords. big :+1: to extracting to concern for reuse wherever we have a password field (or possibly even monkey patching it into audited as a second step to automatically redact passwords and other "sensitive" fields)